specification/maps/data-plane/Search/preview/2021-11-01-preview/geocoding.json

```diff
@@ -0,0 +1,686 @@
+            }
+          }
+        },
+        "geometry": {
+          "$ref": "#/definitions/GeoJSONPoint"
+        },
+        "bbox": {
+          "$ref": "#/definitions/Bbox"
+        }
+      }
+    },
+    "GeocodingResponse": {
+      "description": "This object is returned from a successful Geocoding call",
+      "type": "object",
+      "properties": {
+        "type": {
+          "$ref": "#/definitions/FeatureType"
+        },
+        "features": {
+          "$ref": "#/definitions/Features"
+        },
+        "nextLink": {
+          "$ref": "#/definitions/NextLink"
+        }
+      }
+    },
+    "Address": {
+      "description": "The address of the result",
+      "type": "object",
+      "properties": {
+        "addressLine": {
+          "description": "AddressLine that includes Street Name and Number",
+          "type": "string",
+          "readOnly": true
+        },
+        "locality": {
+          "description": "locality property",
+          "type": "string",
+          "readOnly": true
+        },
+        "neighborhood": {
+          "description": "neighborhood property",
+          "type": "string",
+          "readOnly": true
+        },
+        "adminDistricts": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "name": {
+                "type": "string",
+                "description": "The name for the corresponding adminDistrict field, \nFor adminDistrict[0], this could be full name of state such as Washington, \nFor adminDistrict[1], this could be the full name of the county"
+              },
+              "shortName": {
+                "type": "string",
+                "description": "The short name for the corresponding adminDistrict field, \nFor adminDistrict[0], this could be short name of state such as WA, \nFor adminDistrict[1], this could be the short name of the county"
+              }
+            }
+          }
+        },
+        "postalCode": {
+          "description": "Postal Code property",
+          "type": "string",
+          "readOnly": true
+        },
+        "countryRegion": {
+          "type": "object",
+          "properties": {
+            "ISO": {
+              "description": "ISO of country/region",
+              "type": "string",
+              "readOnly": true
+            },
+            "name": {
+              "description": "name of country/region",
+              "type": "string",
+              "readOnly": true
+            }
+          }
+        },
+        "formattedAddress": {
+          "description": "Formatted Address property",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "MatchCodes": {
+      "type": "array",
+      "items": {
+        "type": "string",
+        "enum": [
+          "Good",
+          "Ambiguous",
+          "UpHierarchy"
+        ]
+      }
+    },
+    "Confidence": {
```

Comment:
Please reach out to Oswaldo Ribas to get more details to add to this field.

Reply:
For now, definition on: https://docs.microsoft.com/en-us/bingmaps/rest-services/locations/location-data

Reply:
updated